src/ixdat/readers/cinfdata_db.py

```diff
@@ -0,0 +1,402 @@
+        y_col = self.group_data[key][:, 1]
+        tstamp = self.group_meta[key]["unixtime"]
+
+        # Create x DataSeries object with appropriate metadata
+        x_series = DataSeries(
+            data=x_col,
+            name=SPECTRUM_METADATA[group_type]["x_name"],
+            unit_name=SPECTRUM_METADATA[group_type]["x_unit_name"],
+        )
+
+        # Create y Field obj with appropriate metadata and x DataSeries as its only axis
+        y_field = Field(
+            data=y_col,
+            name=SPECTRUM_METADATA[group_type]["field_name"],
+            unit_name=SPECTRUM_METADATA[group_type]["field_unit"],
+            axes_series=[x_series],
+        )
+
+        # Create dictionary with spectrum object metadata and x-y Field object
+        spectrum_as_dict = {
+            "sample_name": self.sample_name,
+            "technique": SPECTRUM_METADATA[group_type]["technique"],
+            "field": y_field,
+            "tstamp": tstamp,
+        }
+
+        return spectrum_as_dict
+
+    def add_mass_scans(self):
+        """Add corresponding mass scans to mass_time from 'comment'"""
+        self.mass_scans = True
+        self.measurement_class = MSSpectrum
+        self.grouping_column = "comment"
+        self.token = self.sample_name
+        self.spectrum_list = self.read_spectrums()
+
+        if self.verbose:
+            print(f"Using {self.measurement.time_series[-1]} to find end of experiment")
+            print(
+                "Unixtime end of exp "
+                f"{self.measurement.time_series[-1].data[-1] + self.tstamp}"
+            )
+
+        # Find index of the first spectrum to include as mass scans of this measurement
+        first_index = 0
+        if self.spectrum_list[0].tstamp < self.tstamp:
+            for i, spectrum in enumerate(self.spectrum_list):
+                if spectrum.tstamp < self.tstamp:
+                    first_index = i
+                else:
+                    break
+        # Find index of the last spectrum to include as mass scans
+        last_index = -1
+        measurement_end_time = self.measurement.time_series[-1].data[-1] + self.tstamp
+
+        if self.spectrum_list[-1].tstamp > measurement_end_time:
+            for i, spectrum in reversed(list(enumerate(self.spectrum_list))):
+                if spectrum.tstamp >= measurement_end_time:
+                    last_index = i
+                else:
+                    break
+
+        if self.verbose:
+            print(f"Start and End index of spectrum list: {first_index}, {last_index}\n")
+            print(
+                "Tstamp of first and last spectrum in list: "
+                f"{self.spectrum_list[first_index].tstamp}, "
+                f"{self.spectrum_list[last_index].tstamp}"
+            )
+
+        # Create spectrum series and add it to the measurement
+        spectrums_to_add = self.spectrum_list[first_index:last_index]
+        MSSpectra = SpectrumSeries.from_spectrum_list(spectrums_to_add)
```

Comment:
MSSpectra shouldn't be capitalized as it is an object rather than a class.
Same with SpectroMSMeasurement. And `__add__` method should be called via the `+` operator, not directly.
The lines below should be combined as:
`self.measurement = self.measurement + ms_spectra`

Reply:
done